app/assets/javascripts/require_recaptcha.js

```diff
@@ -0,0 +1,7 @@
+function recaptcha_verified () {
+  document.querySelector('.recaptcha-submit').removeAttribute('disabled')
+}
+
+function recaptcha_expired () {
+  document.querySelector('.recaptcha-submit').setAttribute('disabled', 'disabled')
+}
```

Comment:
I wasn't quite sure where to add this javascript for enabling/disabling the submit button based on the status of the recaptcha.